app/colors/catppuccin.css

```diff
@@ -0,0 +1,200 @@
+[current-mode=insert] #mode-container::before,
+[current-mode=normal] #mode-container::before,
+[current-mode=follow] #mode-container::before,
+[current-mode=explore] #mode-container::before,
+[current-mode=search] #mode-container::before,
+[current-mode=pointer] #mode-container::before,
+[current-mode=visual] #mode-container::before {
+  font-size: 1.5em;
+}
+/*
+* MODE ICONS (Requires a nerd font)
+*/
+[current-mode=command] #mode-container::before {content: "";}
+[current-mode=insert] #mode-container::before {content: "";}
+[current-mode=normal] #mode-container::before {content: "󰇧";}
+[current-mode=follow] #mode-container::before {content: "󰌧";}
+[current-mode=explore] #mode-container::before {content: "󰘎";}
+[current-mode=search] #mode-container::before {content: "";}
+[current-mode=pointer] #mode-container::before {content: "󰆿";}
+[current-mode=visual] #mode-container::before {content: "";}
+/* tabs */
+#tabs > span {padding: 0.25em;font-size: 0.9em;color: var(--tab-foreground);background: var(--tab-background);display: flex;flex: 1;max-width:20em;overflow: hidden;white-space: nowrap;height: 2.5em; border-top: solid .1em transparent;}
+#tabs > ::before {counter-increment: tab-counter 1;content: counter(tab-counter); font-size: 0.85em;}
+#tabs > span > img {margin: auto 0 auto .25em;padding: 0; padding-right: 0.5em;height: 1em;width: 1em;min-height: 1em;min-width: 1em;pointer-events: none;}
+#tabs .visible-tab {background: var(--visible-tab); color: var(--fg); border-top: .1em solid var(--tab-accent);}
+#tabs .pinned {min-width: 3em !important;max-width: 3em;width: 3em; padding: 0.25em;}
+#tabs .pinned > span { display: none; }
+#tabs .pinned > img {padding: 0;margin: auto;height: 1.2em;width: 1.2em;min-height: 1.2em;min-width: 1.2em;}
+/* Nav url/search/command */
+#url {display: inline-block;flex: 1;border: solid .1em var(--url-default);margin: .1em;background: transparent;color: var(--fg);outline: none;font: inherit;min-width: 0;padding: 0.5em 0.4em; line-height: 1;width: 0;cursor: default;}
+/* follow */
+#app #follow .follow-url, #app #follow .follow-inputs-click, #app #follow .follow-inputs-insert, #app #follow .follow-onclick, #app #follow .follow-media, #app #follow .follow-image, #app #follow .follow-other {position: absolute;color: var(--follow-text);padding: 0 .1em; border-radius: 0.2em; font-size: 0.8em;}
+/* mouse related */
+#mode-suggestions {background: transparent;max-height: 0;position: absolute;left: 0.5em;top: 2em;padding-top: .1em;z-index: 37;overflow: hidden;display: block;width: 7em;text-transform: capitalize;display: flex;flex-direction: column;}
+#url-hover {display: none;position: absolute;background: var(--url-hover-bg);color:var(--url-hover-fg);bottom: 0;left: 0;margin: 0;pointer-events: none;padding: .1em;white-space: nowrap;overflow: hidden;text-overflow: ellipsis;width: fit-content;max-width: 80vw;text-align: center;z-index: 4;}
+/* hide logo */
+#logo {display: none}
+
```

Comment:
Please delete one of the final newlines at the end of the file, there are now 2, but 1 is enough.